morf-core/src/main/java/org/alfasoftware/morf/jdbc/AdditionalMetadata.java

```diff
@@ -0,0 +1,27 @@
+package org.alfasoftware.morf.jdbc;
+
+import java.util.Collection;
+
+import org.alfasoftware.morf.metadata.Table;
+import org.apache.commons.lang3.NotImplementedException;
+
+/**
+ * Provides additional metadata.
+ */
+public interface AdditionalMetadata {
+
+  /**
+   * Gets the collection of tables.
+   * @return the collection of {@link Table}
+   */
+  Collection<Table> tables();
```

Comment:
AdditionalMetadata can extend Schema for easier access to things like list of tables, or indexes, etc.

Reply:
Done.